src/resolver.js

```diff
@@ -24,7 +24,6 @@ function resolverFactory(target, options) {
   if (options.before === undefined) options.before = (options) => options;
   if (options.after === undefined) options.after = (result) => result;
   if (options.handleConnection === undefined) options.handleConnection = true;
-  if (options.filterAttributes) options.filterAttributes = resolverFactory.filterAttributes;
```

Comment:
`resolverFactory.filterAttributes` should be changed to true, not sure how i made that mistake.
Setting a default options would not work for everyone who has options defined.